packages/table/src/CheckboxCell/Checkbox.styles.tsx

```diff
@@ -0,0 +1,12 @@
+import { css } from '@leafygreen-ui/emotion';
+
+export const baseStyles = css`
+  // break the first-cell padding. would be cleaner to use a className here, but createUniqueClassname could have performance concerns.
```

Comment:
why performance concerns?

Reply:
Actually, maybe since we only need 2 classNames, this would be fine. For some reason, I was thinking we'd need a unique classname for every first cell.